near-sdk-macros/src/core_impl/info_extractor/arg_info.rs

```diff
@@ -18,12 +18,15 @@ pub enum BindgenArgType {
 /// A single argument of a function after it was processed by the bindgen.
 pub struct ArgInfo {
     /// Attributes not related to bindgen.
+    #[allow(unused)]
```

Comment:
Are these really unused or is it a clippy bug?

Reply:
as far as go-to-references have told me, these are unused. looks like the code was copied from elsewhere, 
 but only some of the fields are used in the context of  `near-sdk-macros`

Reply:
triple-checked these; they are only created inside of `::new` constructors, and not touched by other code thereafter. So, as far as tooling can tell, these are unused